util/thread_local_test.cc

```diff
@@ -67,54 +67,53 @@ TEST_F(ThreadLocalTest, UniqueIdTest) {
   port::Mutex mu;
   port::CondVar cv(&mu);
 
-  // perf_context and iostats_context take 2 ids
-  ASSERT_EQ(IDChecker::PeekId(), 2u);
+  ASSERT_EQ(IDChecker::PeekId(), 0u);
```

Comment:
I don't get why these numbers need to be changed, but I trust you are doing the right thing.